doc/user/data/self_managed/self_managed_operator_compatibility.yml

```diff
@@ -6,6 +6,12 @@ columns:
   - column: "Notes"
 
 rows:
+  - "Materialize Operator": "v25.2.12"
```

Comment:
Should we automate creating these without the "Notes" for when we cut a new self-managed release?